packages/strapi-plugin-content-manager/admin/src/containers/EditViewDataManagerProvider/index.js

```diff
@@ -378,15 +393,15 @@ const EditViewDataManagerProvider = ({
       {
         components: get(allLayoutData, 'components', {}),
       },
-      isCreatingEntry
+      { isCreatingEntry, isDraft: hasDraftAndPublish }
     );
 
     try {
       // Validate the form using yup
       await schema.validate(modifiedData, { abortEarly: false });
 
       // Show a loading button in the EditView/Header.js
-      setIsSubmitting(true);
+      setSending('submit');
```

Comment:
I can't use pending value for each button because all the three buttons will be pending. The loader will be displayed in the submit and publish button. I can set something like `setStatus('submit-pending') & setStatus('publish-pending') & setStatus('unpublish-pending')`

Reply:
That sounds perfect!